core/src/input.rs

```diff
@@ -126,6 +128,7 @@ impl InputManager {
                     InputEvent::KeyDown {
                         key_code: *key_code,
                         key_char: None,
+                        key_location: KeyLocation::Standard,
```

Comment:
We should probably add a TODO for setting these to something reasonable.

Reply:
A good idea, although I think the standard location is very reasonable. It probably won't be left/right because those keys do not have left/right counterparts, it probably won't be numpad because duh, and it won't be dpad because it's AIR only.

Reply:
Done